src/content/docs/alerts-applied-intelligence/transition-guide.mdx

```diff
@@ -1,15 +1,79 @@
 ---
-title: Alerts and applied intelligence transition guide 2022
+title: Alerts and applied intelligence transition guide
 tags:
   - Alerts and Applied Intelligence
   - New Relic Alerts
   - EOL
   - Transition guide
 metaDescription: 'Resources to make a successful transition as new functionalities in New Relic One replace alerting functionalities reaching end of life in 2022.'
+redirects:
+  - /docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/alerts-ai-transition-guide-2022 
 ---
 
 Over the course of several months during 2022, new functionalities in New Relic One will replace alerting functionalities reaching end of life (EOL). We are providing these early notices so you can plan accordingly and make a smooth transition.
 
+## What's changed [#changed]
+
+Starting in late 2021, we've reorganized the navigation UI for alerts and applied intelligence. Although the alerts and applied intelligence navigation has changed pretty dramatically, all the features you use are still there.
+
```

Comment:
I'd love some kind of "why it matters" statement that expands on why we changed the UI. For me, it's very exciting that the UI is now organized in a workflow that clearly states what you can use to solve particular kinds of problems (analyze, detect. etc.).

Reply:
Great suggestion!